src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -1074,6 +1074,86 @@ def get_hosts(self) -> List[orchestrator.HostSpec]:
             )
             ret.append(spec)
         return ret
+    def create_zap_job(self, host: str, path: str) -> None:
+        body = client.V1Job(
+            api_version="batch/v1",
+            metadata=client.V1ObjectMeta(
+                name="rook-ceph-device-zap",
+                namespace="rook-ceph"
+            ),
+            spec=client.V1JobSpec(
+                template=client.V1PodTemplateSpec(
+                    spec=client.V1PodSpec(
+                        containers=[
+                            client.V1Container(
+                                name="device-zap",
+                                image="rook/ceph:master",
+                                command=["bash"],
+                                args=["-c", f"ceph-volume raw list | grep {path} && dd if=/dev/zero of=\"{path}\" bs=1M count=100 oflag=direct,dsync || ceph-volume lvm zap --destroy {path}"],
```

Comment:
```suggestion
                                args=["-c", f"ceph-volume raw list {path} && dd if=/dev/zero of=\"{path}\" bs=1M count=100 oflag=direct,dsync || ceph-volume lvm zap --destroy {path}"],
```

This should be the same. IIRC the `list` supports arguments.

Reply:
Also, I think `count=1` is probably more than enough since the bluestore signature is 60 bytes only.

Reply:
One more thing, `lvm zap` supports OSD FSID and/or OSD ID as a parameter, I wonder if it's safer to use instead of passing the disk, just a thought.